systemverilog/SystemVerilogLexerRules.g4

```diff
@@ -0,0 +1,228 @@
+// Author: Mustafa Said Ağca
+// License: MIT
+
+lexer grammar SystemVerilogLexerRules;
+
+/******************
+	LEXER RULES
+******************/
+
+// 22. Compiler directives
+
+COMPILER_DIRECTIVE
+	: '`' SIMPLE_IDENTIFIER .*? '\r'? '\n' -> skip
```

Comment:
I suggest using `HIDDEN` channel because comments are also sometimes required.

```suggestion
	: '`' SIMPLE_IDENTIFIER .*? '\r'? '\n' -> channel(HIDDEN)
```